lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -31,9 +43,11 @@
   }
 
   &:focus-within {
+    &::after {
+      display: block;
```

Comment:
Why not define the whole thing here rather than having most of it higher up with `display: none`?

Reply:
> Why not define the whole thing here rather than having most of it higher up with `display: none`?

I agree. Defining it here makes it clearer.
aa7d9db5def38dd8cfb54ae2af7e45f49d3f4fba